Views/General/Feed/Card.swift

```diff
@@ -73,7 +73,7 @@ struct Card: View {
                 ForEach(leadingPropertyList, id: \.name) { property in
                     ZStack(alignment: .center) {
                         RoundedRectangle(cornerRadius: 8)
-                            .foregroundColor(property.color ?? .accentColor)
+                            .fill(property.color ?? .accentColor)
```

Comment:
thx for pointing that out, this should be the correct way to fill a shape, changing rest of them in coming commits